xbmc/windowing/osx/WinEventsOSXImpl.mm

```diff
@@ -145,6 +148,7 @@ - (bool)ProcessOSXShortcuts:(XBMC_Event&)event
         return true;
 
       case XBMCK_CTRLF: // CMD-CTRL-f to toggle fullscreen
+      case XBMCK_f: // FN/Globe-f to toggle fullscreen
```

Comment:
if I understand correctly, now cmd+F will also enter fullscreen. Not sure it's right.

still unclear why cmd+ctrl+F doesn't work (if I understood your words right).

btw in Safari menu on macOS 12.6 I also see Fn+F shortcut for Fullscreen, but cmd+ctrl+F still works as well

Reply:
Nice point. This revision was also toggling fullscreen for cmd-f (did not notice that).
Updated in the last revision